react/components/Image.js

```diff
@@ -1,15 +1,39 @@
-import React from 'react'
+import React, { Component } from 'react'
 import PropTypes from 'prop-types'
 import classNames from 'classnames'
+import { withRuntimeContext } from 'render'
+
+const MOBILE_SIZE = 200
```

Comment:
In my desktop the size is 273. And the mobile is 273 too. I think we can only one type like 400. We need to test more how this scale